gateway-protocol/src/main/proto/gateway.proto

```diff
@@ -588,6 +602,13 @@ service Gateway {
   rpc ActivateJobs (ActivateJobsRequest) returns (stream ActivateJobsResponse) {
   }
 
+  /*
+   Registers client to a job stream that will stream jobs back to the client as
+   they are created.
```

Comment:
@koevskinikola Good catch actually! I think, `as they become activatable.` is more precise since we refer `activated` as it is when the worker is currently processing it.